### PR TITLE
Fix/smlog

### PR DESCRIPTION
**Pull request by dgraeber**
### Description:

Modified the credential script for kube, edited the notebook for SageMaker Operators to support SMLogs

### Issue Reference URL

https://github.com/awslabs/aws-orbit-workbench/issues/577

----------
### Do not change content below. Mark applicable check boxes only.

Thank you for submitting a contribution to AWS Orbit Workbench.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a ISSUE associated with this PR?
- [ ] Does your PR title start with ISSUE-XXXX where XXXX is the issue number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [ ] Has your PR been rebased against the latest commit within the target branch (typically 'main')?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under Apache License 2.0?

### CODE REVIEW

File: images/jupyter-user/codeserver_proxy/create_kubeconfig.sh [hunk 1]
```diff
@@ -3,26 +3,30 @@
 env_param='AWS_ORBIT_ENV='
 ep=$(env | grep $env_param)
 cluster_name=orbit-${ep/$env_param/}
+echo $cluster_name
+#orbit-env-r110e-dev
```
Comment (chamcca) [inline images/jupyter-user/codeserver_proxy/create_kubeconfig.sh:7]: this looks like a reference to your local that should be removed

File: samples/notebooks/H-Model-Development/Example-5-SageMaker-on-EKS-xgboost_mnist.ipynb [hunk 2]
```diff
@@ -90,85 +90,63 @@
...
-      "                                          'SourceSecurityGroup': {'GroupName': 'k8s-elb-acbb671f26bdd4ea1b40a9db172b6d30',\n",
-      "                                                                  'OwnerAlias': '616260033377'},\n",
-      "                                          'Subnets': ['subnet-01d0063bf69829484',\n",
-      "                                                      'subnet-03ff5926e434dffad'],\n",
-      "                                          'VPCId': 'vpc-0854c6e3b3afeb178'}},\n",
+      " 'EksPodRoleArn': 'arn:aws:iam::616260033377:role/orbit-env-r110e-dev-lake-user-role',\n",
+      " 'Elbs': None,\n",
       " 'Fargate': True,\n",
-      " 'FinalImageAddress': 'public.ecr.aws/v3o4w1g6/aws-orbit-workbench/jupyter-user:latest',\n",
+      " 'FinalImageAddress': '616260033377.dkr.ecr.us-west-2.amazonaws.com/orbit-env-r110e-dev/jupyter-user:latest',\n",
```
Comment (rb201) [inline samples/notebooks/H-Model-Development/Example-5-SageMaker-on-EKS-xgboost_mnist.ipynb:107]: Is that account number intended to be here?

File: images/jupyter-user/codeserver_proxy/create_kubeconfig.sh [hunk 3]
```diff
@@ -3,26 +3,30 @@
...
 
 user_param='AWS_ORBIT_TEAM_SPACE='
 up=$(env | grep $user_param)
 team_name=${up/$user_param/}
+echo $team_name
+#lake-user
```
Comment (chamcca) [inline images/jupyter-user/codeserver_proxy/create_kubeconfig.sh:13]: another local reference

File: images/jupyter-user/codeserver_proxy/create_kubeconfig.sh [hunk 4]
```diff
@@ -3,26 +3,30 @@
...
 
-#server=$(aws eks describe-cluster --name $cluster_name --query 'cluster.endpoint' | sed "s/\"//g")
-server=""
-name=$(kubectl get secret -oname | grep $team_name-token)
+server=$(aws eks describe-cluster --name $cluster_name --query 'cluster.endpoint' | sed "s/\"//g")
+echo $server
+#https://49B3DA69C6B18379E92C086697CB43CB.gr7.us-west-2.eks.amazonaws.com
```
Comment (chamcca) [inline images/jupyter-user/codeserver_proxy/create_kubeconfig.sh:17]: another local reference

File: images/jupyter-user/codeserver_proxy/create_kubeconfig.sh [hunk 5]
```diff
@@ -3,26 +3,30 @@
...
+
+name=$(kubectl get secret -oname | grep default-editor-token)
+echo $name
+#secret/default-editor-token-l4v9f
```
Comment (chamcca) [inline images/jupyter-user/codeserver_proxy/create_kubeconfig.sh:21]: one more...

File: samples/notebooks/H-Model-Development/Example-5-SageMaker-on-EKS-xgboost_mnist.ipynb (hunk 2 above)
Comment (chamcca) [inline samples/notebooks/H-Model-Development/Example-5-SageMaker-on-EKS-xgboost_mnist.ipynb:107]: these local account references probably shouldn't be here

File: samples/notebooks/H-Model-Development/Example-5-SageMaker-on-EKS-xgboost_mnist.ipynb [hunk 6]
```diff
@@ -90,85 +90,63 @@
...
+      "              'Parameters': {'folder': '/fsx', 'storage': '1200Gi'},\n",
+      "              'Path': None,\n",
+      "              'PluginId': 'fast_fs_lustre'},\n",
       "             {'Module': 'sm-operator',\n",
       "              'Parameters': {},\n",
-      "              'Path': '../plugins/sm-operator/',\n",
+      "              'Path': None,\n",
       "              'PluginId': 'sm-operator'}],\n",
-      " 'Policies': ['orbit-env-r17-demo-lake-user-add-policy'],\n",
+      " 'Policies': ['orbit-env-r110e-dev-demo-lake-user-add-policy'],\n",
```
Comment (chamcca) [inline samples/notebooks/H-Model-Development/Example-5-SageMaker-on-EKS-xgboost_mnist.ipynb:149]: local environment?

File: samples/notebooks/H-Model-Development/Example-5-SageMaker-on-EKS-xgboost_mnist.ipynb [hunk 7]
```diff
@@ -191,19 +169,28 @@
...
+      "               'slug': 'code-server'}],\n",
+      " 'ScratchBucket': 's3://orbit-foundation-dev-env-scratch-616260033377-ahqtil/lake-user',\n",
+      " 'SsmParameterName': '/orbit/env-r110e-dev/teams/lake-user/context',\n",
+      " 'StackName': 'orbit-env-r110e-dev-lake-user',\n",
+      " 'TeamHelmRepository': 's3://orbit-env-r110e-dev-toolkit-616260033377-pnnqrn/helm/repositories/teams/lake-user',\n",
+      " 'TeamKmsKeyArn': 'arn:aws:kms:us-west-2:616260033377:key/2ddbbfc7-7c9b-4c20-aefa-337dbcee9369',\n",
+      " 'TeamSecurityGroupId': 'sg-0fa2efa9012d34247',\n",
+      " 'TeamSsmParameterName': '/orbit/env-r110e-dev/teams/lake-user/team',\n",
+      " 'UserHelmRepository': 's3://orbit-env-r110e-dev-toolkit-616260033377-pnnqrn/helm/repositories/user/lake-user',\n",
+      " 'env_name': 'env-r110e-dev',\n",
```
Comment (chamcca) [inline samples/notebooks/H-Model-Development/Example-5-SageMaker-on-EKS-xgboost_mnist.ipynb:189]: local env and accounts

**Review by chamcca — COMMENTED** [review]
i marked some local env and account spots that i'm not sure we should push to github. there are others, i just stopped marking them. maybe we should clear the output cells from the notebook before committing them?

**Comment by dgraeber** [comment]
> i marked some local env and account spots that i'm not sure we should push to github. there are others, i just stopped marking them. maybe we should clear the output cells from the notebook before committing them?

Yep...I cleared all output and committed

**Review by chamcca — APPROVED** [review]

**Pull request merged**